src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -434,6 +434,12 @@ export default (accessToken, userID, opts) => {
     ),
     usersLoader: gravityLoader("users", {}, { headers: true }),
     userSaleProfileLoader: gravityLoader((id) => `user_sale_profile/${id}`),
+    userAdminNotesLoader: gravityLoader((id) => `user/${id}/admin_notes`),
+    updateAdminNotesLoader: gravityLoader(
```

Comment:
Since this is a `POST`, is this `createAdminNotesLoader` rather than `update`?

Reply:
Yes. Will fix.